lib/game.rb

```diff
@@ -0,0 +1,101 @@
+require 'pry'
+
+class Game
+
+attr_accessor :player_1, :player_2, :board, :the_winner
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+    [0,3,6], # Left
+    [1,4,7], # Middle
+    [2,5,8], #Right
+    [0,4,8], #L Diag
+    [2,4,6]  #R Diag
+  ]
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+      @player_1 = player_1
+      @player_2 = player_2
+      @board = board
+      @the_winner=""
+  end
+
+  def current_player
+    if @board.turn_count % 2 == 0
+      @player_1
+    else
+      @player_2
+    end
+  end
+
+  def won?
+    winning_combo = nil
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```